keyboards/melgeek/z70ultra/rev2/config.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2020 MelGeek <melgeek001365@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xEDED
+#define PRODUCT_ID      0x6575
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    MelGeek
+#define PRODUCT         Z70Ultra
```

Comment:
Should be moved to info.json's usb section.

Reply:
I seriously tested, after the modification can not compile successfully

Reply:
It does, but it's not just drag and drop. 

It should look something like this:
https://github.com/qmk/qmk_firmware/blob/624ab64750ed05e6da46ce3c261e97c7e39c6206/keyboards/boardsource/3x4/info.json#L17-L21